recipes/libcups/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "libcups" %}
+{% set version = "2.2.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/apple/cups/releases/download/v{{ version }}/cups-{{ version }}-source.tar.gz
+  sha256: f58010813fd6903f690cdb0c0b91e4d1bc9e5b9570c28734229ba3ed2908b76c
+
+build:
+  number: 0
+  skip: True   # [not linux]
```

Have you tried also building this for macOS?
```suggestion
  skip: True   # [not unix]
```

---

I have and it doesn't build. This library is part of MacOS SDK anyway